x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/utils.js

```diff
@@ -285,3 +287,30 @@ function jobProperty(job, prop) {
   return job[propMap[prop]];
 }
 
+export function getJobIdUrl(jobId) {
+  // Create url for filtering by job id for kibana management table
+  const settings = {
+    jobId
+  };
+  const encoded = rison.encode(settings);
+  const url = `?mlManagement=${encoded}`;
```

Comment:
is `mlManagement` needed in the URL?
the `jobId` may be enough.

Reply:
I wanted it to be explicit and it gets removed from the url as soon as the filter is cleared.